test/jsapi/index.js

```diff
@@ -9,8 +9,7 @@ var mocha = new Mocha({
   growl: true
 });
 
-// mocha.reporter('spec');
-require('should');
+global.expect = require('expect.js');
```

Comment:
As noted in #2882, `should` is problematic for non-ES5-compliant environments, so we can't run tests using `should` in those environments.

jsapi runs a subset of the unit tests, which now all use `expect` rather than `should`.